website/src/layout.tsx

```diff
@@ -18,31 +20,34 @@ import "./wordpress.css"
 const Layout: React.FC = ({ children }) => {
   return (
     <PreferencesProvider>
-      <Helmet titleTemplate="%s - DAILP" defaultTitle="DAILP">
-        <html lang="en" />
-        <meta charSet="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <body className={themeClass} />
-      </Helmet>
-      <header aria-label="Site Header" id="header" className={css.header}>
-        <div className={css.headerContents}>
-          <MobileNav menuID={2} />
-          <div className={css.contentContainer}>
-            <h1 className={css.siteTitle}>
-              <Link className={css.siteLink} href="/">
-                DAILP
-              </Link>
-            </h1>
-            <span className={css.subHeader}>
-              Digital Archive of Indigenous Language Persistence
-            </span>
+      <LayoutClient>
```

Comment:
Hmm I do think we need `LayoutClient` at this scope but let's move the `Helmet` usage before it since that doesn't call graphql.

```html
<Helmet ...>
</Helmet>

<LayoutClient>
...
```

Reply:
Change has been made!